force_wfmanager/left_side_pane/notification_listener_model_view.py

```diff
@@ -12,6 +12,11 @@ class NotificationListenerModelView(ModelView):
     #: Label to be used in the TreeEditor
     label = Str()
 
+    error_message = Str()
+
+    #: Event to request a verification check on the workflow
+    verify_workflow_event = Event
+
```

Comment:
I suspect you need a handler here too to check if something changes and trigger the event

Reply:
I'm not sure what to put as the check here at the minute. The base notification listeners only have one trait, which is their factory.

Reply:
True. However we will potentially have to deal with validity of the model itself as well. Leave it as is for now, but for example if the model does require a password and it's not there, we will have to mark it as invalid. This makes it even more complex because validity is decided by the model itself...